odata-protocol/11.4 Data Modification.md

```diff
@@ -178,13 +178,22 @@ values beyond those specified in the metadata SHOULD NOT be sent in the
 request body. The service MUST fail if unable to persist all property
 values specified in the request.
 
-Properties computed by the service (annotated with the term
-[`Core.Computed`](https://github.com/oasis-tcs/odata-vocabularies/blob/main/vocabularies/Org.OData.Core.V1.md#Computed),
-see [OData-VocCore](#ODataVocCore)) and properties that are tied to
-properties of the principal entity by a referential constraint, can be
-omitted and MUST be ignored if included in the request.
-
-Properties with a defined default value, nullable properties, and
+Non-insertable properties, as well as dependent properties that are tied to
+non-key properties of the principal entity, SHOULD be omitted from the request body.
```

Comment:
What does "tied to" mean? This here?
```suggestion
Non-insertable properties, as well as properties that depend on 
non-key properties of the principal entity, SHOULD be omitted from the request body.
```

And why only non-key properties? If the key contains a `Country`, is the `CountryName` such a dependent property? Or is it simply a non-insertable property?

Can't we simply say
> Non-insertable properties SHOULD be omitted from the request body.

Because the category of non-insertable properties is already very broad (see below).

Reply:
Providing "foreign-key" values that identify an existing entity establishes a link.

We had corresponding text for updating a link by updating the foreign-key properties, and I've added text for creating a link via foreign-key values.

Reply:
Do you mean denormalized properties? I wouldn't have guessed that from the description. Perhaps
> non-key properties of related entities that appear redundantly in the principal entity